instrumentation/cassandra/cassandra-4.4/testing/src/main/java/io/opentelemetry/testing/cassandra/v4_4/AbstractCassandra44Test.java

```diff
@@ -139,4 +144,23 @@ private static Stream<Arguments> provideReactiveParameters() {
                     "SELECT",
                     "users"))));
   }
+
+  @Override
+  protected CqlSession getSession(String keyspace) {
+    DriverConfigLoader configLoader =
+        DriverConfigLoader.programmaticBuilder()
+            .withDuration(DefaultDriverOption.REQUEST_TIMEOUT, Duration.ofSeconds(0))
+            .withDuration(DefaultDriverOption.CONNECTION_INIT_QUERY_TIMEOUT, Duration.ofSeconds(10))
+            .build();
+    return wrap(
+        CqlSession.builder()
+            .addContactEndPoint(
+                new SniEndPoint(
+                    new InetSocketAddress("localhost", cassandraPort),
+                    "localhost:" + cassandraPort))
```

Comment:
but not sure why network.peer.address becomes ""0:0:0:0:0:0:0:1" instead of "127.0.0.1". i did update networkAttributreExrtractor.

Reply:
> but not sure why network.peer.address becomes ""0:0:0:0:0:0:0:1" instead of "127.0.0.1". i did update networkAttributreExrtractor.

looks like SniEndPoint is using ipv6 for some reason. that's ok, not a problem